common/e2e_config/e2e_config.go

```diff
@@ -88,8 +88,19 @@ type ProductSpec struct {
 	NatsPort                         string            `yaml:"natsPort" env-default:"4222"`
 	NvmeControllerModel              string            `yaml:"nvmeControllerModel" env-default:"Mayastor NVMe controller"`
 	PartialRebuildCpTimeout          string            `yaml:"partialRebuildCpTimeout" env-default:"600s"`
+	PgBenchImage                     string            `yaml:"pgBenchImage" env-default:"postgres:16"`
 	PodLabelKey                      string            `yaml:"podLabelKey" env-default:"app"`
 	PoolCrdName                      string            `yaml:"poolCrdName" env-default:"mayastorpools.openebs.io"`
+	PostgresDatabaseName             string            `yaml:"postgresDatabaseName" env-default:"test"`
```

Comment:
Can you remove all default value from here and move it to product config yaml file. We will be removing all default values from e2e_config.go file later.

Reply:
Done